development/docker-compose.base.yml

```diff
@@ -21,6 +21,8 @@ services:
     depends_on:
       - "postgres"
       - "redis"
+    healthcheck:
```

Comment:
I was just thinking about this... I think we should do this, but it should be in `docker-compose.dev.yml` instead.

Reply:
I can get this moved, but brings up another question, what is the point of the `docker-compose.dev.yml` compared to `docker-compose.yml` within the `development` directory? Is it a bit too many files then, since everything in the directory should be a development file. 

It's just a question that I really don't have an answer for.